java/vsphere-replication/code-examples/src/main/java/dr/restapi/examples/vsphere/replication/VRCodeExamples.java

```diff
@@ -3,39 +3,204 @@
 package dr.restapi.examples.vsphere.replication;
 
 import dr.restapi.examples.apiclient.ApiClient;
-import dr.restapi.examples.apiclient.model.SessionIdData;
-import dr.restapi.examples.apiclient.model.SessionInfo;
+import dr.restapi.examples.apiclient.model.*;
+import dr.restapi.examples.apiclient.model.ConfigureReplicationVmDisk.DestinationDiskFormatEnum;
 import dr.restapi.examples.vsphere.replication.config.Config;
-import dr.restapi.examples.vsphere.replication.libraries.AuthenticationLibrary;
-import dr.restapi.examples.vsphere.replication.libraries.Constants;
-import dr.restapi.examples.vsphere.replication.util.Utils;
+import dr.restapi.examples.vsphere.replication.libraries.*;
+import dr.restapi.examples.vsphere.replication.util.ClientUtils;
+
+import java.util.*;
+
+import static dr.restapi.examples.vsphere.replication.util.ClientUtils.*;
 
 public class VRCodeExamples {
    public static void main(String[] args) {
-      ApiClient apiClient = Utils.getApiClient();
+      ApiClient apiClient = ClientUtils.createApiClient();
 
       VRCodeExamples codeExamples = new VRCodeExamples(apiClient);
-
       codeExamples.runAuthenticationScenario();
+      codeExamples.runConfigureReplicationScenario();
```

Comment:
You first run login, get session and logout (runAuthenticationScenario()) - then call again login, do other work and logout (runConfigureReplicationScenario()).

Reply:
Yes.
Two separate scenarios.